packages/themes/src/ThemeContext/Provider.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import themeContextShape from './shape';
+import HIGLightTheme from "../themes/HIGLightTheme";
+
+export default class Provider extends Component {
```

Comment:
Lets use the new React context API. A polyfill can be found here: https://github.com/jamiebuilds/create-react-context. I used it in production in my last project.